ui/app/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -37,7 +37,7 @@ export default class AccountDetailsModal extends Component {
 
     let exportPrivateKeyFeatureEnabled = true
     // This feature is disabled for hardware wallets
-    if (keyring && keyring.type.search('Hardware') !== -1) {
+    if (keyring && keyring?.type?.search('Hardware') !== -1) {
```

Comment:
I'm guessing that this is what you intended:

```suggestion
    if (keyring?.type?.search('Hardware') !== -1) {
```